lib/uffizzi/config_file.rb

```diff
@@ -61,7 +61,7 @@ def list
         return nil if data.nil?
 
         data.each do |property, value|
-          puts "#{property} - #{value}"
+          Uffizzi.ui.say("#{property} - #{value}")
```

Comment:
I think it's better to form the text for display first and call `Uffizzi.ui.say` just one time